webpack/localization-plugin/README.md

```diff
@@ -0,0 +1,181 @@
+# Localization Plugin for Webpack
+
+## Installation
+
+`npm install @rushstack/localization-plugin --save-dev`
+
+## Overview
+
+This plugin produces webpack bundles that have multiple locales' variants of strings embedded. It also
+has OOB support for RESX files in addition to JSON strings files (with the extension `.loc.json`), including
```

Comment:
OOB is an obscure acronym.  Many people won't know what it means.

Reply:
It would be useful to explain what a "RESX" file is, why someone would want to use it, maybe link to the spec.